lib/csv_helper/exporter.rb

```diff
@@ -3,45 +3,54 @@
 module CsvHelper
   module Exporter
     def export
+      generate(csv_headers)
+    end
+
+    def export_version(number)
```

Comment:
A bit of a nitpick, but can we rename this something like `export_institutions_by_version`, so that it's clear that `institutions` is the only model/csv that uses this?